front/src/schema/NFTCatalogSchema.ts

```diff
@@ -0,0 +1,20 @@
+export interface NFTCardDto {
+    id: number;
+    productId: number;
+    productName: string;
+    nftImageUrl: string;
+    buildDto: BuildDto
+}
+
+export interface BuildDto {
+    rootElement: ShortElementDto;
+    elements: ShortElementDto[];
+}
+
+export interface ShortElementDto {
```

Comment:
Can we have more clear naming here as well? For now, `ShortElement` literally means nothing to me 😃

Reply:
Actually I remembered that we decided to call Gun Component an element. Maybe we can use GunComponent instead, so this is okay until we decide to go for something else.

Reply:
lets discuss :) https://app.clickup.com/t/32h9tyr